libs/judicial-system/formatters/src/lib/formatters.ts

```diff
@@ -12,24 +12,33 @@ const getAsDate = (date: Date | string | undefined | null): Date => {
   }
 }
 
-export function formatDate(date: Date, formatPattern: string): string | null
-export function formatDate(date: string, formatPattern: string): string | null
+export function formatDate(
+  date: Date,
+  formatPattern: string,
+): string | undefined
+export function formatDate(
+  date: string,
+  formatPattern: string,
+): string | undefined
 export function formatDate(
   date: undefined,
   formatPattern: string,
-): string | null
-export function formatDate(date: null, formatPattern: string): string | null
+): string | undefined
+export function formatDate(
+  date: null,
+  formatPattern: string,
+): string | undefined
 
 export function formatDate(
   date: Date | string | undefined | null,
   formatPattern: string,
-): string | null {
+): string | undefined {
```

Comment:
string?

Reply:
All dates are treated as strings because that's how they are sent to the server.